CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Support for config options `[collectd]` and `[opentsdb]` has been removed; use `[[collectd]]` and `[[opentsdb]]` instead.
 * Config option `data-logging-enabled` within the `[data]` section, has been renamed to `trace-logging-enabled`, and defaults to `false`.
 * The keywords `IF`, `EXISTS`, and `NOT` where removed for this release.  This means you no longer need to specify `IF NOT EXISTS` for `DROP DATABASE` or `IF EXISTS` for `CREATE DATABASE`.
+* `max-series-per-database` was added with a default of 1M but can be disabled by setting it to `0`.
```

Comment:
Can we break this out in a separate `Breaking Changes` section?  It should clearly state that if you upgrade and have more than 1M series, you will need to modify your config before upgrading or writes will fail:

```
[data]
max-series-per-database = 0  # 0 to disable
```